jvm-libs/generic/json-rpc/src/main/kotlin/net/consensys/linea/jsonrpc/JsonRpcMessageProcessor.kt

```diff
@@ -50,24 +46,18 @@ private data class RequestContext(
  */
 class JsonRpcMessageProcessor(
   private val requestsHandler: JsonRpcRequestHandler,
-  private val meterRegistry: MeterRegistry,
+  private val metricsFacade: MetricsFacade,
```

Comment:
From Bradley's previous changes, `JsonRpcMessageProcessor` needed to be passed in both `metricsFacade` and `metricsRegistry`, and here I further reduced it to just `metricsFacade`